languages/js/src/Polar.test.ts

```diff
@@ -358,26 +355,22 @@ describe('#loadFile', () => {
   test('throws if two files with the same contents are loaded', async () => {
     const p = new Polar();
     expect(p.loadFile(await tempFile('', 'a.polar'))).resolves;
-    expect(p.loadFile(await tempFile('', 'b.polar'))).rejects.toThrow(
-      PolarFileDuplicateContentError
-    );
+    expect(p.loadFile(await tempFile('', 'b.polar'))).rejects.toThrow();
```

Comment:
@gj couldn't work out the right way to check that the exception thrown has kind `RuntimeError::FileLoading` as specifier in the wasm errors.rs (without manually catching the error and checking it).

Reply:
If you pass a string to `toThrow()` it will do a regex match on the error output

Reply:
Ohh, regex. Nice.

Reply:
Added a regex test check in the latest.